src/datepicker/datepicker.ts

```diff
@@ -139,6 +149,12 @@ export class NgbDatepicker implements OnChanges,
    */
   @Input() startDate: {year: number, month: number};
 
+  /**
+   * An event fired when navigation happens and currently displayed month changes.
+   * The payload is the new month we're navigating to.
+   */
+  @Output() navigate = new EventEmitter<{year: number, month: number}>();
```

Comment:
On the one hand additional info won't hurt, on the other hand people can save previous month on their own, so not sure.

Would be smth like:
```ts
export interface NgbDatepickerNavigationEvent {
  current: {year: number, month: number};
  next: {year: number, month: number};
}
```